app.js

```diff
@@ -35,24 +38,34 @@ checker.results.failed = function(node) {
 	this.parent.updateStats(node);
 };
 
-// ////////////////////////////////////////////////////////////////////////////////////
-// STATUS
 let Status = function(parent, index) {
 	this.parent = parent;
 	this.tag = document.createElement("span");
 	this.tag.textContent = ' WAIT: 🕑 - ';
-	this.up = false;
 };
 
 Status.prototype.check = function() {
-	const gatewayAndScriptHash = this.parent.gateway.replace(":hash", SCRIPT_HASH);
-	const rnd = encodeURIComponent(Math.random());
-	const src = `${gatewayAndScriptHash}?i=${this.parent.index}&rnd=${rnd}#x-ipfs-companion-no-redirect`;
-	let script = document.createElement('script')
+	let gatewayAndScriptHash = this.parent.gateway.replace(":hash", SCRIPT_HASH);
+
+	// we set a random number as a url parameter, to try to prevent content caching
+	// is it right ? ... do you know a better way ? ... does it always work ?
+	let rnd = Math.random();
```

Comment:
Unix timestamp is bit better as it is guaranteed to be unique across page loads:
```suggestion
	let rnd = new Date()).getTime();
```